kivy/metrics.py

```diff
@@ -217,8 +223,13 @@ def fontscale(self):
 
         if platform == 'android':
             from jnius import autoclass
-            PythonActivity = autoclass('org.renpy.android.PythonActivity')
-            config = PythonActivity.mActivity.getResources().getConfiguration()
+            if USE_SDL2:
+                # This activity name is temporary, it will need to be changed later
+                NewPythonActivity = autoclass('net.inclem.android.NewPythonActivity')
```

Comment:
Yep, I'll definitely change it later, I started this way to be clear it wasn't final :)